easybuild/easyblocks/generic/intelbase.py

```diff
@@ -260,31 +262,39 @@ def install_step(self, silent_cfg_names_map=None, silent_cfg_extras=None):
         if silent_cfg_names_map is None:
             silent_cfg_names_map = {}
 
-        # license file entry is only applicable with license file or server type of activation
-        # also check whether specified activation type makes sense
-        lic_file_server_activations = [ACTIVATION_LIC_FILE, ACTIVATION_LIC_SERVER]
-        other_activations = [act for act in ACTIVATION_TYPES if act not in lic_file_server_activations]
-        lic_file_entry = ""
-        if self.cfg['license_activation'] in lic_file_server_activations:
-            lic_file_entry = "%(license_file_name)s=%(license_file)s"
-        elif not self.cfg['license_activation'] in other_activations:
-            raise EasyBuildError("Unknown type of activation specified: %s (known :%s)",
-                                 self.cfg['license_activation'], ACTIVATION_TYPES)
-
-        silent = '\n'.join([
-            "%(activation_name)s=%(activation)s",
-            lic_file_entry,
+        if self.cfg['licensed']:
```

Comment:
maybe it's better to use `if self.runtime_license:` here, which is set to something not `None` in `prepare_step` in case there is a runtime license?